.github/workflows/release-from-main.yml

```
@@ -52,12 +52,15 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Get CI Bot Token
-        id: ci_bot_token
-        uses: tibdex/github-app-token@v2
+      - name: Get release-lane token
+        id: release_lane_token
+        uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)